data/hai missions.txt

```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
 	on complete
 		payment 100000
 		dialog `Instead of landing in the spaceport where they can be seen, you drop Anaya and Touhar off as close as you can to where they will be staying. The house is a few kilometers away, but they assure you that they can make it on their own. They pay you <payment> and thank you for bringing them this far.`
+
+
+
+mission "Nanachi 1"
+	name "Escort Nanachi to Cloudfire"
+	description "Teach Nanachi, a Hai who just bought her first ship, the basics of being a starship captain."
+	source "Giverstone"
+	destination "Cloudfire"
+	deadline
+	to offer
+		"net worth" > 20000000
+		"combat rating" > 1200
+		"random" > 85
+	npc accompany save
+		personality escort timid
+		government "Hai"
+		ship "Heavy Shuttle" "Acorn Chaser"
+	on visit
+		dialog "You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for them to arrive in this star system."
+	on offer
+		conversation
+			"As you are making your way through the spaceport you notice a worried-looking Hai staring at the local job board and back at some sort of document they are holding."
+			choice
+				"(Approach the Hai.)"
+				"(Leave them alone.)"
+					decline
+			`The hai is a bit startled, but quickly calms down, "Sorry, you caught me by surprise. I've just bought my first ship and need to find some jobs to do, but nobody I've talked to wants to trust a beginner captain. I just don't know what to do anymore, I've been working for so long to qualify for this loan, but I'm running out of time and I'll never be able to pay it like this."`
```

Comment:
Again, tab in within the backticks. Also, Hai isn't capitalized here. There should also be a full stop after `but quickly calms down` 

```
`	The Hai is a bit startled, but quickly calms down. "Sorry, ..."`
```